projects/plugins/jetpack/class.jetpack.php

```diff
@@ -824,6 +824,7 @@ public function configure() {
 
 		if ( ! $this->connection_manager ) {
 			$this->connection_manager = new Connection_Manager( 'jetpack' );
+			Automattic\Jetpack\ConnectionUI\Admin::init();
```

Comment:
Since this is still a work in progress, what do you say we add a filter to conditionally enable initialization of the Connection UI, so we can disable it by default until it's ready?